src/Auctions/Auction.sol

```diff
@@ -622,7 +623,7 @@ contract Auction is Governance2Step, ReentrancyGuard {
         require(paymentAmount != 0, "zero amount");
         require(available(address(order.sellToken)) != 0, "zero available");
         require(order.feeAmount == 0, "fee");
-        require(order.partiallyFillable, "partial fill");
+        require(!order.partiallyFillable, "partial fill");
```

Comment:
* partial takes allow for the potential for an auction to remain open until expiry if the remaining sell amount is too low
* while an auction is live, the price continues to decline
* we may have an issue if more sell token becomes available during the auction: the new `available` amount could now be taken at a low price. I'm unsure that this is a possibility but just noting it in case you do. for example, having a check in the strategy preventing it from sending reward tokens to the auction contract while an auction for that token is active should protect us 
* if we are protected, then allowing partial takes from either cowswap or any other taker makes sense.